src/App.js

```diff
@@ -1,16 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
 
 const App = () => {
+  console.log(chatMessages);
+  const [chatsMessages, setChatState] = useState(chatMessages);
+  const changeLike = (id) => {
+    const newData = chatsMessages.map((chat) => {
+      if (chat.id === id) {
+        chat.liked = !chat.liked;
+        return chat;
+      }
+      return chat;
+    });
+    return setChatState(newData);
+  };
+  const sumLikes = () => {
+    const likedTrue = chatsMessages.filter((chat) => {
+      return chat.liked === true;
+    });
+    return likedTrue.length;
```

Comment:
I like this logic in a method so that you can invoke the method when line 30 renders